src/routes/create_user.ts

```diff
@@ -0,0 +1,46 @@
+import express from "express"
+import { Client } from "../entities/Client";
+const router = express.Router();
+
+router.post('/api/client', async(req,res)=>{
+    const {
+        user_id,
+        username,
+        name,
+        password
+        
+    } = req.body;
+    const clientFind = await Client.findOne({ where: { user_id } });
+    const clientFind1 = await Client.findOne({ where: { username } });
```

Comment:
make this variable name more efficient and meaningful to be readable . 
- also why are you calling the database twice ? to check the same user ?

Reply:
I created this for testing ,so I will edit the variable name and consider merging them.